script.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
```

Comment:
I guess this file shouldn't be included in the PR

Reply:
Right 🤦  good catch!